common/app/common/TrailsToRss.scala

```diff
@@ -131,9 +131,10 @@ object TrailsToRss extends implicits.Collections {
         .filterNot(_.config.excludeFromRss)
         .flatMap(_.all)
         .filter{
-        case _: LinkSnap => false
-        case _ => true}
-        .distinctBy(_.id)
+          case _: LinkSnap => false
```

Comment:
Not a problem but fyi I think `collect` may be slightly more idiomatic (and shorter):

```scala
.collect {
    case a: LinkSnap => a
}
```